tools/protocols/src/Delegate.ts

```diff
@@ -124,10 +129,10 @@ export class Delegate {
   }
 
   async getQuotedOrMaxAvailable(
-    senderToken,
-    senderAmount,
-    signerToken,
-    signerAmount
+    senderToken: string,
+    senderAmount: string,
```

Comment:
on line 142, balance.lt(senderAmount) will this still work if the input is a string? same with the other math calculations that are now converted using bigNumerify.

Reply:
Yes, the argument for function `lt` is of type BigNumberish (BigNumber | string | number | Arrayish)